maps/torch/datums/uniforms_fleet.dm

```diff
@@ -3,24 +3,20 @@
 	departments = COM
 
 	utility_under = /obj/item/clothing/under/solgov/utility/fleet/command
-	utility_extra = list(
-				/obj/item/clothing/under/solgov/utility/fleet/combat/command,
-				/obj/item/clothing/head/beret/solgov/fleet/command,
-				/obj/item/clothing/head/ushanka/solgov/fleet,
-				/obj/item/clothing/suit/storage/hooded/wintercoat/solgov/fleet,
-				/obj/item/clothing/head/soft/solgov/fleet,
-				/obj/item/clothing/gloves/thick/duty/solgov/cmd
-			)
+	utility_extra = list(/obj/item/clothing/head/beret/solgov/fleet/command,
+						 /obj/item/clothing/head/ushanka/solgov/fleet,
+						 /obj/item/clothing/suit/storage/hooded/wintercoat/solgov/fleet,
+						 /obj/item/clothing/head/soft/solgov/fleet,
+						 /obj/item/clothing/gloves/thick/duty/solgov/cmd,
+						 /obj/item/clothing/under/solgov/utility/fleet/polopants/command,
+						 /obj/item/clothing/suit/storage/jacket/solgov/fleet/command)
```

Comment:
Is there a reason for the uh, super intense indentation? This seems to be something occurring throughout the file. If there's no reason for it, I'd request you tone it down a little and being it more in line with other formatting.